src/forms/progressive-profiling-popup/index.jsx

```diff
@@ -56,14 +55,13 @@ const ProgressiveProfilingForm = () => {
 
   useEffect(() => {
     let countryCode = null;
-    if (languageCookieValue) {
-      countryCode = languageCookieValue;
+    if (countryCookieValue) {
+      countryCode = countryCookieValue;
     } else {
       countryCode = authContextCountryCode;
```

Comment:
Will this be undefined by default? So what happens if country code is not available in MFE context call?

Reply:
The country field will be populated with UNDETECTED if there is not country code in cookie and MFE context call.